hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipUpdateTest.java

```diff
@@ -819,6 +832,82 @@ public void testListenerRegistrationWhileANewMemberJoining() {
         assertOpenEventually(entryAddedLatch);
     }
 
+    @Test
+    public void testDistributedObjectsAccurateAfterBatchedJoinHandling() throws InterruptedException {
+        // Prepare config
+        List<String> membersList = new ArrayList<>(10);
+        for (int k = 0; k < 10; k++) {
+            membersList.add("127.0.0.1:" + (5701 + k));
+        }
+        Config config = smallInstanceConfigWithoutJetAndMetrics();
+        config.getNetworkConfig().setPort(5701);
+        config.setClusterName("myCluster").getNetworkConfig().getJoin().getTcpIpConfig().setMembers(membersList);
```

Comment:
started to fail on master after enabling tcp config and started to pass with this fix:

```suggestion
        JoinConfig joinConfig = config.setClusterName("tcpIpConfig")
                .getNetworkConfig().getJoin();
        TcpIpConfig tcpIpConfig = joinConfig.getTcpIpConfig();
        tcpIpConfig.setEnabled(true)
                .setMembers(membersList);
```

Reply:
Failing on master is the expectation (as there is no fix on master) - apply your changes here breaks the test, as the cluster name no longer matches. I have updated the test to explicitly enable the `TcpIpConfig` as your change does. Please let me know if you're having any trouble with the latest test 👍

Reply:
Not sure why i changed cluster name, some copy-paste error possibly, my main intention was to enable TcpIpConfig.

it looks like ok now, thx.